manifests/postgresql-operator-default-configuration.yaml

```diff
@@ -25,8 +25,11 @@ configuration:
     pod_role_label: spilo-role
     cluster_labels:
         application: spilo
+    # inherited_labels:
```

Comment:
so these are not the actual labels but rather label keys ? but the actual "key:value" pair that forms a label should be inherited ?

Reply:
i see, Can you please add this paragraph to the operator conf docs ?

Reply:
Sure, I've updated the [`docs/administrator.md`](https://github.com/stang/postgres-operator/blob/add-inherited-labels/docs/administrator.md#add-cluster-specific-labels) with a short explanation and a snippet of config that illustrate it.